mllib/src/main/scala/org/apache/spark/mllib/feature/IDF.scala

```diff
@@ -55,7 +55,7 @@ class IDF @Since("1.2.0") (@Since("1.2.0") val minDocFreq: Int) {
       seqOp = (df, v) => df.add(v),
       combOp = (df1, df2) => df1.merge(df2)
     ).idf()
-    new IDFModel(idf)
+    new IDFModel(idf._1, idf._2, idf._3)
```

Comment:
To make this more readable, assign the output of treeAggregate to a tuple of names, so you don't have to write ._1, etc

Reply:
Done